.lycheeignore

```diff
@@ -0,0 +1,6 @@
+# API Domain stubs used as url partials in the api clients
+api.rev.ai
+getBaseUrl
```

Comment:
Checking your failed build logs - https://github.com/revdotcom/revai-node-sdk/runs/5796182915?check_suite_focus=true#step:4:73- should this be lowercased to work?
```suggestion
getbaseurl
```